config/application.rb

```diff
@@ -9,7 +9,9 @@ class Application < Rails::Application
     config.encoding = "utf-8"
     config.filter_parameters += [:password]
     config.active_support.escape_html_entities_in_json = true
-
+    config.action_view.field_error_proc = Proc.new { |html_tag, instance| 
```

Comment:
For multiline block - I would prefer `do` / `end` rather than `{}`. Anyway It looks like you want your own errors handling?